### DISCUSSION TITLE
How are Cigale's model parameter priors calculated?

### DISCUSSION THREAD

**Discussion by junhopen** [mboquien/cigale#103] [Q&A] [answered]
Hello！I want to know how cigale will calculate the result when I specify multiple values ​​for a parameter. For example, when I set metallicity to 0.004, 0.02, 0.05, will the prior of the model be a Gaussian distribution of these three values?

**Comment by mboquien** [comment, marked as answer]
First, it is important that free parameters are given several values. It is not unusual for have grids of hundreds of thousands or even millions of models (CIGALE does the Cartesian product of all free parameters to compute all the possible combinations). When several metallicities are given, as for any other parameter, the models are computed and fitted to the observations. Then they are weighted based on the likelihood (∝exp(-χ²/2)). To answer your question, the prior will be made of just these 3 discrete values.

**Reply by junhopen** [reply]
Thanks for your answer！
